WordPressKit/Domains/DomainsServiceRemote+AllDomains.swift

```diff
@@ -0,0 +1,133 @@
+import Foundation
+
+extension DomainsServiceRemote {
+
+    // MARK: - API
+
+    public func fetchAllDomains(params: AllDomainsEndpointParams? = nil, completion: @escaping (AllDomainsEndpointResult) -> Void) {
+        let path = self.allDomainsEndpointPath(params: params)
+        self.wordPressComRestApi.GET(path, parameters: nil) { result, _ in
+            do {
+                switch result {
+                case .success(let result):
+                    let data = try JSONSerialization.data(withJSONObject: result, options: [])
+                    let decoder = JSONDecoder()
+                    decoder.dateDecodingStrategy = .iso8601
+                    let result = try decoder.decode(AllDomainsEndpointResponse.self, from: data)
+                    completion(.success(result.domains))
+                case .failure(let error):
+                    throw error
+                }
+            } catch let error {
+                completion(.failure(error))
+            }
+        }
+    }
+
+    private func allDomainsEndpointPath(params: AllDomainsEndpointParams?) -> String {
```

Comment:
The request is missing the `no_wpcom=true` query. From the project spec:
> To retrieve the list of domains we’ll make an authenticated GET request to https://public-api.wordpress.com/rest/v1.1/all-domains?no_wpcom=true&resolve_status=true&locale={LOCALE}

Reply:
Good catch. Resolved in f6f8461df348031f5eba645a488463f2f878daaa